openproblems/tasks/denoising/methods/knn_smoothing.py

```diff
@@ -8,16 +8,17 @@
 
 # the decay and t parameter give knn smoothing behavior
 def _knn_smoothing(adata, solver):
-    import knn-smooth
-    
-    smoothX = knn-smooth.knn_smoothing(adata.X)
+    import knn - smooth
+
+    smoothX = knn - smooth.knn_smoothing(adata.X)
     adata.X = smoothX
     return adata
 
 
 def knn_smoothing(adata, test=False):
     return _knn_smoothing(adata)
 
+
 @method(

```

Comment:
The decorator needs to go directly before the function, otherwise it doesn't decorate the function

Reply:
got it, put the @method decorator above the function